sdk/samples/radio_example_tx/main.c

```diff
@@ -7,18 +7,23 @@
 #include "radio.h"
 #include "rftimer.h"
 #include "scm3c_hw_interface.h"
+#include "scum.h"
 
 // Number of for loop cycles between Hello World messages.
 // 700000 for loop cycles roughly correspond to 1 second.
 #define NUM_CYCLES_BETWEEN_TX (1000000UL)
-#define NUM_CYCLES_BETWEEN_PACKET (100UL)
+#define NUM_CYCLES_BETWEEN_PACKET (1000UL)
 #define TX_PACKET_LEN (64UL)
 
 void tx_endframe_callback(uint32_t timestamp);
-
 uint8_t packet[TX_PACKET_LEN] = {0};
 
+uint8_t fine_code = 0;
+uint8_t mid_code = 0;
```

Comment:
Should also all be declared static:
```suggestion
static uint8_t packet[TX_PACKET_LEN] = {0};

static uint8_t fine_code = 0;
static uint8_t mid_code = 0;
```